web/pages/meetings/1st_workshop_2020_at_bryn_mawr/index.md

```diff
@@ -0,0 +1,148 @@
+title:
+author: Erik T. Everson
+
+# 1st PlasmaPy Workshop
+#### April 20-22, 2020
+#### Bryn Mawr College, Pennsylvania
+<br/>
+[Overview](#Overview)<br/>
+[Agenda](#Agenda)<br/>
+[Registration]()<br/>
+[Travel & Housing](#Travel-Housing)<br/>
+[Travel Funding for Students](#Travel-Funding-for-Students)<br/>
+[Web Participation](#Web-Participation)<br/>
+[Support & Acknowledgements](#Support-Ack)
+
+### <a name="Overview"></a> Overview
+
+PlasmaPy's first mini-conference.  The conference is setup as a 3 day meeting for 
+PlasmaPy developers and the PlasmaPy community. The first day (4/20) is 
+designated for the PlasmaPy Development Team and the remaining two days (4/21-4/22) are 
+designed for community interaction.  Goals of the meeting are to (1) further develop
+PlasmaPy's roadmap, (2) inform the broader PlasmaPy community of PlasmaPy's objectives 
+and direction, (3) demonstrate PlasmaPy's capabilities to the community, and 
+(4) receive valuable feedback from the community.<br/><br/>
+
+### <a name="Agenda"></a> Agenda
+
+---
+**NOTE**
+
+As the meeting approaches, please check the agenda for updates.  As presentation material
+becomes available we will post links for slides and such.
+___
+
+
+_**Day 1 - April 20, 2020**_
+
+Day 1 is intended for the PlasmaPy committee and internal topics.  Attendees will be
+restricted to the PlasmaPy committee and developers.  We'll address topics related to 
+package development, community outreach, and education.
+
+| Time | Section | Title | Speaker/Leader |
+| --- | --- | --- | --- |
+| 08:30 - 09:00 | 0  | **ARRIVAL & COFFEE** |  |
+| 09:00 - 09:15 | 1  | Welcome: Introduction & Overview |  |
+| 09:15 - 09:35 | 2  | Roadmap: Overview & Current Projections |  |
+| 09:35 - 10:35 | 3  | Roadmap: Formulary Development |  |
```

Comment:
I don't think we'll need as much time for formulary development as for simulations and experiments.

Reply:
Oh yeah, I originally had 1 hr for every roadmap session and, I guess, this one was missed when I changed to 45 min sessions.

Reply:
Looks like we get a slightly longer lunch. I don't have a problem with that!😁